### PR TITLE
docs: migration from Docusaurus to GitHub wiki

### PR DESCRIPTION
**Pull request by Neudrino**
## Summary

- This is part of the effort from https://github.com/magma/magma/issues/9848.
- To avoid having diverging documentation the content was copied over and linked in the appropriate places for a seamless transition.
- Please see **https://github.com/magma/magma/wiki/Contributing-Code** for the migration target.
- While the migration is not completed with this, it is a first valuable step and imho finalizable.

### CODE REVIEW

**Review by electronjoe — APPROVED** [review]
Thank you for leaving behind the migration breadcrumbs.

In my assessment, this migration looks to be appropriate given our objectives.

**Comment by themarwhal** [comment]
@Neudrino merge conflict

**Comment by themarwhal** [comment]
Should we migrate these links as well? https://github.com/magma/magma/pull/10953#issuecomment-998953067

**Comment by Neudrino** [comment]
> Should we migrate these links as well? [#10953 (comment)](https://github.com/magma/magma/pull/10953#issuecomment-998953067)

Good point. Did not think of them so far. Will have a look here.

File: .github/workflows/pr_bot.yml [hunk 1]
```diff
@@ -29,13 +29,13 @@ jobs:
             ### More info
 
             Please take a moment to read through the Magma project's
-            - [Contributing Conventions](https://docs.magmacore.org/docs/next/contributing/contribute_conventions) for norms around contributed code
+            - [Contributing Conventions](https://github.com/magma/magma/wiki/Contributing-Code-Conventions) for norms around contributed code
 
             If this is your first Magma PR, also consider reading
             - [Developer Onboarding](https://docs.magmacore.org/docs/next/contributing/contribute_onboarding) for onboarding as a new Magma developer
```
Comment (Neudrino) [inline .github/workflows/pr_bot.yml:35]: This points to the docusaurus intentionally still, as it is not migrated yet.

**Comment by Neudrino** [comment]
* Migrated your, @themarwhal , new addition to Github wiki and resolved the merge conflict.
* Added the changes needed for the Github actions to display the correct links.

**Review by themarwhal — APPROVED** [review]

**Review by ildikov — APPROVED** [review]
Hi,

I'm sorry for just getting to this one, I missed it over the holidays and it remain under my radar. :(

Thanks @Neudrino for the cleanup work here and finalizing the migration of these sections so we don't end up with two versions of the same document.

Even though we will keep polishing this content in the Contributor Guide, it is in the stage now that it is preferred to make the changes there. The changes in this PR look good to me.

Thanks,
Ildikó

**Comment by Neudrino** [comment]
Rebased on master again, because the Pipelines seemed stuck.

**Review by electronjoe — APPROVED** [review]

**Comment by themarwhal** [comment]
Hi @Neudrino ! Just curious, should I not be seeing this page anymore with this PR merged? https://docs.magmacore.org/docs/next/contributing/contribute_ci_checks

**Comment by Neudrino** [comment]
> Hi @Neudrino ! Just curious, should I not be seeing this page anymore with this PR merged? https://docs.magmacore.org/docs/next/contributing/contribute_ci_checks

Yes, @themarwhal , that would also have been my assumption. This is very curious. I do not know, why this is not the case. :man_shrugging: 

Guessing: There is probably some step, which generates the Docusaurus pages from the repository. However, I do not know where this conversion runs or how often. Wild guessing would be, that it runs on the machine, the Docusaurus is hosted on?

But still, as to why this would not update is unclear to be so far.

**Comment by themarwhal** [comment]
Hello @jbryce! Do you have insight into why this doc change is not reflected? 

As an example, this page https://docs.magmacore.org/docs/next/contributing/contribute_ci_checks should now look like this.
Screen Shot 2022-01-31 at 10 19 07 AM

**Comment by ildikov** [comment]
Since the Docusaurus is versioned, I assume the changes might need to be "released" first to be visible?

**Comment by Neudrino** [comment]
Thanks @ildikov ! However, the link @themarwhal posted was to the `next` version of the Docusaurus, which should always be up to date with the `docs/readmes` folder of the [master branch of the repository](https://github.com/magma/magma/tree/master/docs/readmes), afaik.

**Comment by Neudrino** [comment]
This seems to be a more general issue with the infrastructure here, as [this newer commit](https://github.com/magma/magma/commit/b118787ab24396f8f5548c718316a812829748c9) is also _not_ reflected [at the respective page](https://docs.magmacore.org/docs/next/orc8r/deploy_install#initialize-terraform).

FYI @jbryce @themarwhal @jaredmullane

**Comment by Neudrino** [comment]
I created https://github.com/magma/magma/issues/11350 for this issue. It might be caused by https://github.com/magma/magma/commit/83d68a2cdfb182eb0804d7fa1d43f158d5b2f5f5.

**Pull request merged**